README.md

```diff
@@ -18,12 +20,53 @@ Some layers include the corresponding OTel language SDK for the Lambda. This all
 * ### [.NET + Collector Lambda Layer](dotnet/README.md)
 * ### [Collector Lambda Layer](collector/README.md)
 
-
 ## FAQ
+
 * **What exporters/recievers/processors are included from the OpenTelemetry Collector?**
     > You can check out [the stripped-down collector's imports](https://github.com/open-telemetry/opentelemetry-lambda/blob/main/collector/lambdacomponents/default.go#L18) in this repository for a full list of currently included components.
 * **Is the Lambda layer provided or do I need to build it and distribute it myself?**
     > This repository does not provide pre-build Lambda layers. They must be built manually and saved in your AWS account. This repo has files to facilitate doing that. More information is provided in [the Collector folder's README](collector/README.md).
 
 ## Design Proposal
+
 To get a better understanding of the proposed design for the OpenTelemetry Lamda extension, you can see the [Design Proposal here.](docs/design_proposal.md)
+
+## Features
+
+The following is a list of features provided by the OpenTelemetry layers.
+
+### Custom Context Propagation
+
+Context can be propagated through various mechanisms (e.g. http headers (APIGW), message attributes (SQS), ...). In some cases, it may be required to pass a custom context propagation extractor in lambda through configuration, this feature allows this through Lambda instrumentation configuration.
+
+### Disable AWS Context propagation via environment variable
+
+The context propagation when Active Tracing is enabled uses the `_X_AMZN_TRACE_ID` environment variable present in the Lambda runtime. To disable this behaviour, implementations have added the `OTEL_LAMBDA_DISABLE_AWS_CONTEXT_PROPAGATION` environment variable which can be set to `true`.
+
+### Semantic conventions
+
+The Lambda language implementation follows the semantic conventions specified in the OpenTelemetry Specification.
+
+### Auto instrumentation
+
+The Lambda layer includes support for automatically instrumentation code via the use of instrumentation libraries.
+
+### Support matrix
+
+The table below captures the state of various features and their levels of support different runtimes.
+
+| Feature                    | Node | Python | Java | .NET | Go   | Ruby |
```

Comment:
We don't have Go, .NET, or Ruby layers.  They use the stock collector-only layer.  Should this refer to the Lambda instrumentations in each language instead?

Reply:
I think there was discussion from @tsloughter about adding a ruby layer based on the current splunk implementation.

I would expect the other layers to be empty until they're implemented, but i see what you're saying, why have them at all if they're not supported yet.

Reply:
It's not so much that there are empty layers that will eventually not be empty.  Go will never have a layer that is more than the bare collector as the instrumentation needs to be compiled into the application.  I think .NET is similar.

Reply:
Fair enough, in that case I would capture the collector as a separate "feature" of the layer, to make clear that those other supported runtimes weren't forgotten. Otherwise I would expect users to see the dotnet layer in this repository and ask "what features does it support"